doc/content/getting-started/what-is-tts/_index.md

```diff
@@ -12,7 +12,15 @@ Welcome to {{% tts %}}! In this section, we help you to understand how to use {{
 
 {{% tts %}} is an enterprise grade LoRaWAN network server, built on an [open source](https://github.com/TheThingsNetwork/lorawan-stack) core. {{% tts %}} allows you to build and manage LoRaWAN networks on your own hardware or in the cloud.
 
-To get started running {{% tts %}} on your own, see the [installation section]({{< ref "/getting-started/installation" >}}). To inquire about a worry-free hosted solution, [get in touch with us](mailto:info@thethingsindustries.com).
+### {{% tts %}} Deployments
+
+{{% tts %}} comes in many different flavors to meet different requirements. For all available deployments, see [Get {{% tts %}}]({{< ref "download" >}}). The most common deployments are:
```

Comment:
People can't download Cloud. That is Open Source only which is not listed here. Maybe add a fourth option `**Open Source**` with the download link?

Reply:
Download describes all the deployments, the title is a relic.

<img width="1495" alt="Bildschirmfoto 2021-07-08 um 14 33 39" src="https://user-images.githubusercontent.com/6963436/124922427-c763f680-dff9-11eb-81e4-58a0b03e9670.png">